demos/demos_databases_apis/databricks_pyspark/graphistry-notebook-dashboard.ipynb

```diff
@@ -56,76 +59,53 @@
-       "datasetInfos": [],
-       "metadata": {},
-       "removedWidgets": [],
-       "type": "ansi"
-      }
-     },
-     "output_type": "display_data"
+     "output_type": "stream",
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001B[43mNote: you may need to restart the kernel using dbutils.library.restartPython() to use updated packages.\u001B[0m\nRequirement already satisfied: graphistry in /local_disk0/.ephemeral_nfs/cluster_libraries/python/lib/python3.10/site-packages (0.34.3)\nRequirement already satisfied: numpy in /databricks/python3/lib/python3.10/site-packages (from graphistry) (1.23.5)\nRequirement already satisfied: requests in /databricks/python3/lib/python3.10/site-packages (from graphistry) (2.28.1)\nRequirement already satisfied: pandas in /databricks/python3/lib/python3.10/site-packages (from graphistry) (1.5.3)\nRequirement already satisfied: setuptools in /databricks/python3/lib/python3.10/site-packages (from graphistry) (65.6.3)\nRequirement already satisfied: squarify in /local_disk0/.ephemeral_nfs/cluster_libraries/python/lib/python3.10/site-packages (from graphistry) (0.4.4)\nRequirement already satisfied: typing-extensions in /databricks/python3/lib/python3.10/site-packages (from graphistry) (4.4.0)\nRequirement already satisfied: palettable>=3.0 in /local_disk0/.ephemeral_nfs/cluster_libraries/python/lib/python3.10/site-packages (from graphistry) (3.3.3)\nRequirement already satisfied: pyarrow>=0.15.0 in /databricks/python3/lib/python3.10/site-packages (from graphistry) (8.0.0)\nRequirement already satisfied: packaging>=20.1 in /databricks/python3/lib/python3.10/site-packages (from graphistry) (22.0)\nRequirement already satisfied: python-dateutil>=2.8.1 in /databricks/python3/lib/python3.10/site-packages (from pandas->graphistry) (2.8.2)\nRequirement already satisfied: pytz>=2020.1 in /databricks/python3/lib/python3.10/site-packages (from pandas->graphistry) (2022.7)\nRequirement already satisfied: idna<4,>=2.5 in /databricks/python3/lib/python3.10/site-packages (from requests->graphistry) (3.4)\nRequirement already satisfied: certifi>=2017.4.17 in /databricks/python3/lib/python3.10/site-packages (from requests->graphistry) (2022.12.7)\nRequirement already satisfied: charset-normalizer<3,>=2 in /databricks/python3/lib/python3.10/site-packages (from requests->graphistry) (2.0.4)\nRequirement already satisfied: urllib3<1.27,>=1.21.1 in /databricks/python3/lib/python3.10/site-packages (from requests->graphistry) (1.26.14)\nRequirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.8.1->pandas->graphistry) (1.16.0)\n\u001B[43mNote: you may need to restart the kernel using dbutils.library.restartPython() to use updated packages.\u001B[0m\n"
```

Comment:
I like to do `pip install -q` to quiet, or just full clear this output, as not super helpful